spec/integration/mentioning_spec.rb

```diff
@@ -1,13 +1,21 @@
 require "spec_helper"
 require "requests_helper"
+require "integration/federation/federation_helper"
```

Comment:
Can you try to write this tests without the federation helper? When you don't use `remote_user_on_pod_b` and `remote_user_on_pod_c` (which add stubs on the callbacks which you don't need), you also don't need to use  `allow_callbacks` for the tests (no stubs == everything allowed with call original, which is what you want here).

Have a look at `remote_raphael`.